celeri/config.py

```diff
@@ -80,6 +80,41 @@ class Config(BaseModel):
     Reasonable values might be in the range of 0.01 to 10.
     """
 
+    segment_slip_rate_bound_weight: float = 100.0
+    """Weight for enforcing slip rate bounds at segments.
+
+    This is used in `solve_sqp2` to enforce slip rate bounds softly.
+    """
+
+    segment_slip_rate_hard_bounds: bool = True
+    """Enforce hard slip rate bounds at segments.
+
+    This should be disabled when using soft slip rate bounds
+    via `segment_slip_rate_bound_sigma`.
```

Comment:
But this is `True` and `segment_slip_rate_bound_sigma` is `3.0`. :thinking:

Why is this the case, and should it be enforced in the code?